recipes/rpkmforgenes/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: "rpkmforgenes"
+  version: "1.0.1"
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/danielramskold/S3_species-specific_sequencing
+  git_rev: 1.0.1
+  md5: foo
```

Comment:
Doh! I mistakenly in magit pushed to master and then ended up with the wrong version in the feature branch. Should be fixed by now.